src/streamlink/plugins/pluzz.py

```diff
@@ -1,216 +1,132 @@
 import logging
 import re
-import sys
-import time
+from datetime import datetime
+from urllib.parse import urlparse
 
-from streamlink.plugin import Plugin, PluginArgument, PluginArguments, pluginmatcher
-from streamlink.plugin.api import validate
-from streamlink.stream import DASHStream, HDSStream, HLSStream, HTTPStream
-from streamlink.stream.ffmpegmux import MuxedStream
+from isodate import LOCAL as LOCALTIMEZONE
+
+from streamlink.plugin import Plugin, PluginError, pluginmatcher
+from streamlink.plugin.api import useragents, validate
+from streamlink.stream import DASHStream, HLSStream
+from streamlink.utils.url import update_qsd
 
 log = logging.getLogger(__name__)
 
 
-@pluginmatcher(re.compile(r'''
-    https?://(
-        (?:www\.)?france\.tv/.+\.html
-        |
-        www\.(ludo|zouzous)\.fr/heros/[\w-]+
+@pluginmatcher(re.compile(r"""
+    https?://(?:
+        (?P<francetv>(?:www\.)?france\.tv)/
```

Comment:
1. you don't use this groups in `pluginmatcher`, they could be removed
2. https://github.com/streamlink/streamlink/blob/master/docs/plugin_matrix.rst needs an update